python/k4FWCore/IOSvc.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2014-2024 Key4hep-Project.
+#
+# This file is part of Key4hep.
+# See https://key4hep.github.io/key4hep-doc/ for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from Configurables import IOSvc as IO
+import os
+
+class IOSvc:
+
+    def __init__(self, *args, **kwargs):
+
+        self._svc = IO(**kwargs)
+
+    def __getattr__(self, attr):
+        return getattr(self._svc, attr)
+
+    def __setattr__(self, attr, value):
+        print(f'IOSvc: {attr} {value}')
+        if attr == '_svc':
+            super().__setattr__(attr, value)
+            return
+
+        if attr == 'input':
+            if isinstance(value, str):
+                value = [value]
+        if attr == 'output':
+            if os.path.dirname(value) and not os.path.exists(os.path.dirname(value)):
+                os.makedirs(os.path.dirname(value))
```

Comment:
Is there a reason to create a python wrapper instead of putting this in a C++ class directly?

Reply:
I think this is also addressed in #170 already.

Reply:
#170 does that when using `PodioOutput`. Then whether to have it in C++ or python, well in python it's a one liner, compared to #170 for example and it's something that runs at most once. It could also be part of the podio backends

Reply:
My concern was that the python wrapper will have different behavior and it's not related to make it work or be nicer in python. I think the intent was to have python as configuration layer but keep the logic in C++

#170 does also error handling